### PR TITLE
[APR] Add pool without gauges to the api

### PR DESCRIPTION
**Pull request by Rawallon**
This PR aims to allows pools without gauge to be instanciated as a Pool class, for free we get the URL for a single pool

[Example URL from 80 20](https://tools-monorepo-4wfey4afb-bleu-fi.vercel.app/apr/pool/ethereum/0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014?startAt=09-14-2023&endAt=09-28-2023)

[API Example URL from 80 20](https://tools-monorepo-4wfey4afb-bleu-fi.vercel.app/apr/pool/ethereum/?startAt=09-14-2023&endAt=09-28-2023&poolId=0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014&)

image

### CODE REVIEW

**Comment by Rawallon** [comment]
Currently we can display pools without gauges

image

File: apps/balancer-tools/src/app/apr/(utils)/getRelativeWeight.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ export const getPoolRelativeWeight = withCache(
     time: number = Date.now() / 1000,
   ) {
     const gaugeAddress = new Pool(poolId).gauge?.address;
-    if (!gaugeAddress) throw new Error(`No gauge found for pool ${poolId}`);
+    if (!gaugeAddress) return 0;
+    // if (!gaugeAddress) throw new Error(`No gauge found for pool ${poolId}`);
```
Comment (luizakp) [inline apps/balancer-tools/src/app/apr/(utils)/getRelativeWeight.ts:22]: should remove this comment?

File: apps/balancer-tools/src/app/apr/api/(utils)/validate.ts [hunk 2]
```diff
@@ -80,9 +81,11 @@ export const QueryParamsSchema = z
   .object({
     poolId: OptionalNullableString.refine(
       (poolId) =>
-        !poolId ||
         POOLS_WITH_LIVE_GAUGES.some(
           (g) => g.id.toLowerCase() === poolId?.toLowerCase(),
+        ) ||
+        POOLS_WITHOUT_GAUGES.some(
+          (p) => p.id.toLowerCase() === poolId?.toLowerCase(),
```
Comment (luizakp) [inline apps/balancer-tools/src/app/apr/api/(utils)/validate.ts:88]: I wasn't able to run on dev 🤔 

image

**Review by luizakp — CHANGES_REQUESTED** [review]
Let's check why this is not opening the homepage before merging it

File: apps/balancer-tools/src/app/apr/api/(utils)/validate.ts (hunk 2 above)
Comment (Rawallon) [reply apps/balancer-tools/src/app/apr/api/(utils)/validate.ts:88]: That's odd, cause last deploy's [URL as example](https://tools-monorepo-mrc7mvl0k-bleu-fi.vercel.app/apr/pool/ethereum/0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014?startAt=09-21-2023&endAt=09-22-2023) it seems to work, did u manage to fix this on your end?

**Comment by luizakp** [comment]
@Rawallon only ethereum pools are returning

**Pull request merged**